index.html

```diff
@@ -142,20 +191,72 @@ <h2 class="sub-title">Project name goes here</h2>
                     </div>
                   </div>
           </div>
+    <div class="projects-d-container">
+
+      <div class="projects-d-1">
+        
+      </div>
+      <div class="projects-d-2">
+        
+      </div>
+      <div class="projects-d-3">
+        
+      </div>
+
+      <div class="projects-d-4">
+        <div class="projects-d-41">
+          
+        </div> 
+        <div class="projects-d-42">
+         
+        </div> 
+        
+      </div>     
+      
+      <div class="projects-d-6">
+        <div class="projects-d-61">
+          
+        </div>
+        <div class="projects-d-62">
```

Comment:
- [ ] You have done great so far. :+1: :+1:  But there is one issue pointed out by the previous reviewer you missed.
- [ ] As pointed out by the reviewer, you have a vertical scroll bar in your portfolio which is not in the Figma design. I'll suggest you go through your CSS and fix the error. See the video below too.

- - VIDEO REFERENCE:



https://github.com/hydaline-code/Portfolio_mobilefirst/assets/107632780/4d00733e-4d98-4b82-acc8-5712556a25e6

Reply:
Great done by fixing some of the issues pointed before 👍🏼  , and to fix this one you can add ` overflow-x: hidden;` to your html element this will prevent your page from scrolling horizontaly